packages/jest-circus/src/utils.ts

```diff
@@ -163,6 +164,7 @@ export const callAsyncCircusFn = (
   testContext: Circus.TestContext | undefined,
   asyncError: Circus.Exception,
   {isHook, timeout}: {isHook?: boolean | null; timeout: number},
+  testOrHook: Circus.TestEntry | Circus.Hook,
```

Comment:
I wonder if we can pass _just_ `testOrHook` and not `fn` and `asyncError` separately... Not for this PR, but maybe something we can land on master

Reply:
Please let me know if that's something you'd like to have

Reply:
Sure, I can take that up next 😄